src/api/github-client.ts

```diff
@@ -237,4 +250,21 @@ export class CheGithubClient {
 
     return sortedSemanticTags.map(tag => tag.data)
   }
+
+  private async getCommitData(repo: string, commitId: string) {
+    const owner = this.getOwnerByRepo(repo)
+    const commitDataResponse = await this.octokit.repos.getCommit({ owner, repo, ref: commitId })
+    if (commitDataResponse.status !== 200) {
+      throw new Error(`Failed to get commit data from the repository '${repo}'. Request: ${commitDataResponse.url}, response: ${commitDataResponse.status}`)
+    }
+    return commitDataResponse.data
+  }
+
+  async getCommitDate(repo: string, commitId: string): Promise<string> {
```

Comment:
I think it is better to pass `che-inclubator/chectl` to get rid of `getOwnerByRepo`

Reply:
Anyway, in that case, we have to have a function to parse `org/repo`. No profit as for me.